packages/plugin-core/src/test/suite-integ/SyncCommand.test.ts

```diff
@@ -390,6 +396,71 @@ suite("workspace sync command", function () {
       });
     });
   });
+
+  describe("WHEN there are tracked, uncommitted changes", () => {
```

Comment:
one failure mode: user stashes changes, git pull causes rebase. what happens in this case? can we add a test?